_infra/helm/response-operations-ui/templates/deployment.yaml

```diff
@@ -135,6 +135,8 @@ spec:
             value: {{ .Values.notify.requestCreateAccountTemplate }}
           - name: NOTIFY_REQUEST_PASSWORD_CHANGE_TEMPLATE
             value: {{ .Values.notify.requestPasswordChangeTemplate }}
+          - name: NOTIFY_CHANGE_ACCOUNT_DETAILS_TEMPLATE
```

Comment:
Kinda picky, but the value should more closely match the key name.  So this should be:
`NOTIFY_UPDATE_ACCOUNT_DETAILS_TEMPLATE`

I won't comment against everything that uses this value ,but everything that uses this value needs to be updated as well!